regex-assembly/932280.ra

```diff
@@ -0,0 +1,8 @@
+##! Please refer to the documentation at
+##! https://coreruleset.org/docs/development/regex_assembly/.
+
+##! Bash Brace Expansion
+##! Detects the following patterns which are common in Unix shell scripts and one-liners
+
+
+\{[,\-0-9a-z]+\}
```

Comment:
The characters within the braces are even less limited. Commas can be quoted, for example. I think this would be more appropriate: `\{[^},]*,[^}]*\}`.

Reply:
This is too broad, it's prone to FPs don't you think.

Reply:
Yes, probably. Maybe we need two rule then, one for PL 1 and one for PL 3 maybe.

Reply:
I think we should come up with combination of your regex and mine, the only difference is it'll be limited like mine.